esmvaltool/cmorizers/data/formatters/datasets/esacci_sst.py

```diff
@@ -30,17 +30,21 @@
    20201204-roberts_charles: written.
```

Comment:
When running `esmvaltool data format ESACCI-SST` I noticed that only the NCL script is called and not this one.

Reply:
This is because we first check for NCL and then for python. Now the old NCL version is removed so it should not be a problem